### ISSUE TITLE
Accessibility: announce comment level with talkback

### ISSUE THREAD

**Issue by shortwavesurfer2009** [dessalines/jerboa#642]
** Jerboa Version ** 0.0.34

The comment level should be announced when using talkback. As an example, redreader announces 

"Level 1 comment Author: username # up vote. # hours, # mins ago." Or "Level 2 comment Author: username # down votes # hour, ## mins ago.".

There is a setting under the accessibility settings that reads "Use concise language when possible, Make accessibility labels less verbose to make navigation more efficient for experienced Redditors". These would appear like this 

"L1 username # up # hours, ## mins ago." Or "L2 username # down # hour, ## mins ago."

**Comment by kuroi-usagi** [comment]
That is not a bug but a feature request.

**Comment by shortwavesurfer2009** [comment]
> That is not a bug but a feature request.

I cant see how to edit the label. Its arguable as to if its a bug or not because a sighted user can see what level they are at where a talkback user cannot, therefore making it a bug as that is unintended

**Comment by dessalines** [comment]
Stale issue. If someone wants to work on this I can re-open.
